var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -234,5 +248,26 @@ def cmake_args(self):
         else:
             # we rely on the fftw-api@3
             options.append('-DGMX_FFT_LIBRARY=fftw3')
+            if '^amdfftw' in self.spec:
+                options.append('-DGMX_FFT_LIBRARY=fftw3')
+                options.append(
+                    '-DFFTWF_INCLUDE_DIRS={0}'.
+                    format(self.spec['amdfftw'].headers.directories[0])
+                )
+                options.append('-DFFTWF_LIBRARIES={0}'.
+                               format(self.spec['amdfftw'].libs.joined(';')))
+
+        if '%aocc' in self.spec:
+            options.append('-DGMX_CYCLE_SUBCOUNTERS:BOOL=OFF')
+            options.append('-DCMAKE_C_FLAGS=-Ofast -march=znver2 -ffast-math')
```

Comment:
Isn't this `-march` flag machine specific? Spack already injects optimization flags automatically. I don't think we have `%aocc` support yet in `archspec`, but it should be hard to add. @alalazo